packages/python/src/jarvis_protocol/__init__.py

```
@@ -1,3 +1,1400 @@
+    if forbidden_field:
+        return protocol_error(
+            "forbidden_host_private_field",
+            {
+                "field": forbidden_field,
+                "reason": "Protocol records MUST NOT expose host-private fields.",
+            },
+        )
+    return None
+
+
+__all__ = [
```

Ruff (0.15.20)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)